From 26bfc35711730c2ed32526b5a33c52a39869808f Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Tue, 14 Mar 2017 10:19:49 -0700
Subject: [PATCH] DOCKER-1016 Upgrade docker to bunyan 1.8.8 to better handle
 long log lines Reviewed by: Josh Wilsdon <josh@wilsdon.ca> Approved by: Josh
 Wilsdon <josh@wilsdon.ca>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 6c6a822..d75c9e5 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "assert-plus": "1.0.0",
     "async": "0.9.0",
     "docker-registry-client": "3.2.6",
-    "bunyan": "1.8.5",
+    "bunyan": "1.8.8",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "fwrule": "git+https://github.com/joyent/sdc-fwrule.git#d1174be",
     "imgmanifest": "git+https://github.com/joyent/node-imgmanifest.git#91e5d80",
-- 
2.21.0

